plugins/importtext_procedure.cc

```diff
@@ -77,7 +77,7 @@ ImportTextConfigDescription::ImportTextConfigDescription()
     addField("named", &ImportTextConfig::named,
              "Row name expression for output dataset. Note that each row "
              "must have a unique name.",
-             SqlExpression::parse("lineNumber()"));
+             SqlExpression::parse("dataFileUrl() + '.' + lineNumber()"));
```

Comment:
File path can be very long.  I am concerned that a fair portion of the imported dataset memory footprint will be comprised of the repeated file path.  It seems to me that this is a problem specific to the merge dataset.  Why not change the merge dataset to prefix/suffix the row names of the datasets?  Something like`[]-[12]`for rows in the right dataset and `[729]-[]` for rows in the left dataset.